packages/editor/src/editor/sidebar/fields/table-field/TableField.tsx

```diff
@@ -76,18 +75,6 @@ export const TableField = <TData extends object>({ label, data, onChange, column
     }
   });
 
-  useEffect(() => {
```

Comment:
What was the idea of this useEffect? Because it triggered an endless update error if the inscription was opened with an empty row...

Reply:
If I remember correctly, the idea here was that when a table is edited and then exited, leaving no row selected, it should check for any empty rows and delete them immediately if found. This was my attempt to prevent saving empty rows that serve no purpose. Therefore, the useEffect checks whether a row is selected; if not, all empty rows are removed. I have already applied this logic in the Inscription. Unfortunately, I never noticed that this could lead to an endless loop, as there should never be a table with an empty row using this logic.

Reply:
Yes I thought that it is something like this...
However I thought a bit about this here, And I'm not sure if this makes sense here. Because maybe someone wants to have an empty option in the select?
But I quickly check if I can fix instead of remove it